clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -138,6 +156,15 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
     }
   }
 
+  private activateJob(TitusClient provider, Job job, boolean disable) {
```

Comment:
I know this is groovy but there's no return type on this method.

Reply:
it's not `disable` since you flipped the logic
```suggestion
  private activateJob(TitusClient provider, Job job, boolean inService) {
```

Reply:
> it's not `disable` since you flipped the logic

yes agree , fixed it.